pkg/policy/policies.go

```diff
@@ -205,6 +217,173 @@ func (ps *Policies) Delete(id int) error {
 	return nil
 }
 
+func (ps *Policies) computeEventsStates(cfg config.Config) {
```

Comment:
All logic related only to Events will be relocated to the events pkg. This is a draft.

Reply:
We've decided that events states are a Policies thing. So, they will continue in policy pkg.